### PR TITLE
Fit 434 Require Snapshot feature specs in build process testing of Intake App

### PR DESCRIPTION
**Pull request by SergiiRedchuk**
### Jira Story

https://osi-cwds.atlassian.net/browse/FIT-434

## Description
Added a Jenkins phase to run only tests for Snapshot (without randomly failing tests for screenings)

### CODE REVIEW

**Comment by SergiiRedchuk** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -142,16 +151,17 @@ def build() {
 
 def incrementTag() {
   stage('Increment Tag') {
+    curStage = 'Increment Tag'
```
Comment (AVandana-OSI) [inline Jenkinsfile]: Is this `curStage` is required to add to each stage?

Comment (edgibbs) [reply Jenkinsfile]: It's used for the slack message, probably could be refactored, but probably out of scope for this build.

Comment (SergiiRedchuk) [reply Jenkinsfile]: yes, is is used in the catch block inside slackNotification, I just added its assignment where it was missing to make the Jenkinsfile more consistent, no refactoring this time

**Comment by SergiiRedchuk** [comment]
> Code Client is broken, please fix it.

I don't have permission to code client. Getting 403.
cc @edgibbs

**Pull request merged**